src/utils/generateUsageSnippets.js

```diff
@@ -0,0 +1,32 @@
+import { FEATURE_SETS_TAB, FEATURE_VECTORS_TAB } from '../constants'
+
+export const generateUsageSnippets = pageTab => {
+  if (pageTab === FEATURE_SETS_TAB) {
+    return {
+      title: 'Get offline features for training:',
+      code: `features = [
+"<feature set>.*",
+]
+vector = FeatureVector(features=features)
+resp = get_offline_features(
+vector, entity_rows=<entity>, entity_timestamp_column="<entity timestamp column"
```

Comment:
Almost :) Just needs to add 4-space indentation in two places here:

```suggestion
    "<feature set>.*",
]
vector = FeatureVector(features=features)
resp = get_offline_features(
    vector, entity_rows=<entity>, entity_timestamp_column="<entity timestamp column"
```